build.gradle

```diff
@@ -29,11 +30,11 @@ application {
     mainClass = "${moduleName}.Main"
 }
 
-// Uncomment below to simulate running from the CLI
-//run {
-//    standardInput = System.in
-//    args = ["-cli", "path/to/config"]
-//}
+// Required to accept a name and tiebreak interactively via CLI
+run {
+    standardInput = System.in
+    args = ["-cli", "path/to/config"]
+}
```

Comment:
What are the ramifications of leaving this uncommented now? Before it was a quick way for me to swap in having the `run` commend launch the CLI instead of the GUI.

How can we easily switch to this mode now using Gradle?

Reply:
Ah, I didn't read this too closely. The `standardInput` is needed and has no ramifications, best I can tell. the fact that `-cli` was not `--cli` meant the `args` were ignored. Will fix and leave `args` commented.